libPhoneNumberGeocoding/NBPhoneNumberOfflineGeocoder.h

```diff
@@ -0,0 +1,132 @@
+//
+//  NBPhoneNumberOfflineGeocoder.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <libPhoneNumberGeocoding/NBGeocoderMetadataHelper.h>
+#import <libPhoneNumberGeocoding/NBPhoneNumberOfflineGeocoder.h>
+
+@class NBPhoneNumber;
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NBGeocoderMetadataHelper *_Nonnull (^NBGeocoderMetadataHelperFactory)(NSNumber *countryCode,
+                                                                              NSString *language);
+
+@interface NBPhoneNumberOfflineGeocoder : NSObject
+
+- (instancetype)initWithMetadataHelperFactory:(NBGeocoderMetadataHelperFactory)factory;
```

Comment:
Can you add a documentation comment here?

Reply:
Yup, Done.